framework/web/Request.php

```diff
@@ -261,6 +261,14 @@ class Request extends \yii\base\Request
         'Front-End-Https' => ['on'], // Microsoft
     ];
 
+    /**
+     * @var array list of the key prefixes used to obtain HTTP headers from $_SERVER variable.
+     */
+    public $headerPrefixes = [
```

Comment:
I'm not sure if we want to expose that. We were quite fine with one prefix so far, and the second one looks like an exception from the rule only.